config/feeds_config_v2.json

```diff
@@ -41669,42 +41730,6 @@
         ]
       }
     },
-    {
-      "id": "1000007",
```

Comment:
We should mark this feed as deprecated and not remove it, as otherwise we risk a new unrelated feed to push data to the same contract address.